README.md

```diff
@@ -16,51 +16,49 @@ AWS offers EC2-optimized macOS AMIs for developer use. These AMIs include this T
 | EC2 System Monitor for macOS | For collecting system monitoring CloudWatch metrics | Cask | ec2-macos-system-monitoring | 
 
 ## Installing the Tap
-The AMIs provided by EC2 come with Homebrew "pretapped" with this Tap. To manually install the Tap, run the the `brew tap` command:
+The AMIs provided by AWS come with Homebrew "pretapped" with this Tap. To manually install the Tap, run the the `brew tap` [command](https://docs.brew.sh/Taps):
```

Comment:
Since the introduction begins "**AWS** _offers_ ... AMIs..." it's more parallel to refer to the "AMIs _provided by_ **AWS**". Also, that's consistent with Amazon's own internal usage:
```
$ aws --region us-west-2 ec2 describe-images --image-ids ami-0b3c31b0c66a46614 --query 'Images[].Description'
[
    "macOS Big-Sur 11.4 AMI built by Amazon Web Services"
]
$
```
It's not built by EC2, it's built by AWS as part of the EC2 service offering.